### PR TITLE
Improve common data configuration check

### LINKED ISSUE #10498: Do not show Data Library tab when no Data Library is available
Also make sure that it is not offered when creating a new map.

Most likely map tutorial shouldn't be shown in this case

cc @juanignaciosl

### PR DESCRIPTION
**Pull request by juanignaciosl**
This fixes #10498.

This was _really_ simple, I thought that it was not implemented but actually an small check improvement is enough for this be easier.

@javitonino CR this, please.

cc @rporres

### CODE REVIEW

File: app/services/visualization/common_data_service.rb [hunk 1]
```diff
@@ -22,7 +22,7 @@ def self.load_common_data(user, controller)
       end
 
       def self.configured?
-        !Cartodb.config[:common_data].nil?
+        !Cartodb.get_config(:common_data, 'base_url').nil?
```
Comment (javitonino) [inline app/services/visualization/common_data_service.rb]: Maybe `Cartodb.get_config(:common_data, 'base_url').present?` is clearer.

**Comment by juanignaciosl** [comment]
@javitonino `present?` refactor done.

**Comment by javitonino** [comment]
:shipit:

**Comment by javitonino** [comment]
Those unrelated NEWS :laughing:

**Pull request merged**